nixos/modules/services/networking/adguardhome.nix

```
@@ -44,6 +52,18 @@ in
       '';
     };
 
+    settings = mkOption {
+      type = types.attrs;
+      default = { };
+      description = ''
+        Configurations for AdGuard Home. On start, these options are merged into
+        the configuration file, overwriting changes to the same options made
+        with the web interface.
+
+        See https://github.com/AdguardTeam/AdGuardHome/wiki/Configuration#configuration-file for options.
+      '';
```

```suggestion
      description = ''
        AdGuard Home configuration. Refer to
        <link xlink:href="https://github.com/AdguardTeam/AdGuardHome/wiki/Configuration#configuration-file"/>
        for details on supported values.
        
        <note><para>
          On start, these options are merged into
          the configuration file, overwriting changes to the same options made
          with the web interface.
        </para></note>
      '';
```

- Users love clickable links :heart:
- It is probably worth making this note standout so users understand exactly what is going to happen ahead of time.